### ISSUE TITLE
Combat Site - Class 3 - Refinery Defense Mission - Unable to start again.

### ISSUE THREAD

**Issue by TheAgeLord** [RavienGaming/Reforged-Eden-2#366]
### Reforged Eden 2 Build Number

23

### Game mode

Dedicated Server

### Tested in vanilla

Not tested in vanilla or not applicable

### Description of issue

After my first attempt at a Class 3 combat site (the Refinery Defense quest), I failed. Now, whenever I try it again in a different system, the quest skips some stages and moves straight to the combat phase, but no enemy ships spawn. I found a player who hadn’t done the Refinery quest before; when he spoke with the NPC, the mission started correctly, and ships appeared. However, we failed again. Now I’m not sure if the quest is bugged for him too, but I can’t complete it solo. I've tried disabling and re-enabling the mission and speaking to the NPC multiple times, but the enemy ships still won’t spawn.

![image_2024-10-28_190146825](https://github.com/user-attachments/assets/df704f02-34e2-4af4-8029-6768659392ba)

### POI Name

Ally Refinery

### Playfield Name

Every Combat Site class 3

### Playfield Class

Combat Site class 3

### Star Name

all?

### Star Class

all?

### Mission Name

EdenSpace_POI_CombatT3_2

### Mission Objective

Defeat Raider Vessels

### Game Seed

I am not server admin sorry.

**Comment by RavienGaming** [comment]
The mission automatically resets itself when you speak to a NPC in a different combat site and start the mission again.
If it is not doing this then that would be a game bug that we cannot fix unfortunately.

One thing you can try is manually deactivate the mission, then manually activate a different mission and then try again at a new combat site.

**Comment by TheAgeLord** [comment]
Tried both, deactivating the mission, and new combat sites, both the mission resets in wrong chapter of the quest, skipping steps may be the reason why ships are not spawning...The only solution I can still try is find another player that could still start the quest, so the ships spawn and win the quest.

**Comment by RavienGaming** [comment]
Yes if you do it along with another player that should finally "fix" it for you. Unfortunately the PDA can be quite buggy and that's a game issue. Hope that works for you.

**Comment by redwrench** [comment]
Stumbled upon this because I was having the exact same issue. I was able to fix it with the `pda clrchap XXX` command.

You'll have to search for the specific chapter number for "Trigger: EdenSpace_POI_CombatT3_2" using `pda ls`. Mine was 272, but yours might be different if running other mods/scenarios.

Edit: You WILL need admin perms to run this command, so I'm not sure what to do about that on a public server.

**Comment by RavienGaming** [comment]
Yeah unfortunately this is probably due to several PDAop functions not working correctly on servers. In SP this mission gets reset whenever you start it again, but in MP I have a feeling it does not. It's something that needs to be reported to Eleon to fix.
